config/v1/types_scheduling.go

```diff
@@ -28,20 +28,16 @@ type Scheduler struct {
 }
 
 type SchedulerSpec struct {
-	// DEPRECATED: the scheduler Policy API has been deprecated and will be removed in a future release.
-	// policy is a reference to a ConfigMap containing scheduler policy which has
-	// user specified predicates and priorities. If this ConfigMap is not available
-	// scheduler will default to use DefaultAlgorithmProvider.
-	// The namespace for this configmap is openshift-config.
-	// +optional
-	Policy ConfigMapNameReference `json:"policy,omitempty"`
 	// profile sets which scheduling profile should be set in order to configure scheduling
 	// decisions for new pods.
 	//
 	// Valid values are "LowNodeUtilization", "HighNodeUtilization", "NoScoring"
 	// Defaults to "LowNodeUtilization"
 	// +optional
 	Profile SchedulerProfile `json:"profile,omitempty"`
+	// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles.
```

Comment:
What does `certain` profiles mean? 
```suggestion
	// profileCustomizations contains configuration for modifying the default behavior of certain profiles.
```